packages/domain-picker/src/utils/index.ts

```diff
@@ -34,19 +38,23 @@ interface DomainSuggestionsVendorOptions {
 	isSignup?: boolean;
 	isDomainOnly?: boolean;
 	isPremium?: boolean;
-	flowName?: 'link-in-bio';
+	flowName?: NEWSLETTER_FLOW | LINK_IN_BIO_FLOW;
```

Comment:
shouldn't these be `typeof NEWSLETTER_FLOW | typeof LINK_IN_BIO_FLOW`

Reply:
the types are defined above https://github.com/Automattic/wp-calypso/pull/67395/files#diff-6d74f7c73d47e0ff89367a7ff72f691cab489e2e15048421dd400c6adb300c16R4 but I can inline them as well